jupyterlab_git/handlers.py

```diff
@@ -521,6 +538,7 @@ def setup_handlers(web_app):
         ("/git/log", GitLogHandler),
         ("/git/pull", GitPullHandler),
         ("/git/push", GitPushHandler),
+        ("/git/remote/add", GitRemoteAddHandler),
```

Comment:
I think this could be made a bit more RESTful. I agree with using `/git/remote` as the entity but perhaps we could use HTTP verbs to define our actions, which would also make it extensible for further remote-related use-cases.

- POST `/git/remote`: Add a new remote
- GET `/git/remote`: List/get remotes
- DELETE `/git/remote`: Remove a remote

Reply:
I would definitely like to have a more RESTful API (see #495 for discussion). The trouble is that this requires the repository path to be part of the endpoint - especially for action without body (like GET and usually DELETE). So to stay coherent with the current API, I would rather keep the current logic. And let the change be done in another PR.